.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 /.idea
 Cargo.lock
+db_path
```

Comment:
I think we should clean every temporary files created by test cases, maybe we should create another issue.